linux_os/guide/services/mail/postfix_client/postfix_client_configure_mail_alias/tests/correct.pass.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 # packages = postfix
 
-echo "root: system.administrator@mail.mil" > /etc/aliases
+echo "root: change_me@localhost" > /etc/aliases
```

Comment:
In addition I would prefer to explicitly set the variable value using the variables test scenario header to prevent unwanted drifts of the value.